skyvern/forge/sdk/api/llm/schema_validator.py

```
@@ -0,0 +1,182 @@
+        The validated and filled extraction result
+    """
+    if schema is None:
+        LOG.debug("No schema provided, returning extraction result as-is")
+        return extraction_result
+
+    LOG.info("Validating and filling extraction result against schema")
+
+    try:
+        filled_result = fill_missing_fields(extraction_result, schema)
+        LOG.info("Successfully validated and filled extraction result")
+        return filled_result
```

Ruff (0.14.5)

Consider moving this statement to an `else` block

(TRY300)